src/components/NavBar/index.js

```diff
@@ -1,200 +1,214 @@
 import {
   AppBar,
-  Avatar,
-  Box,
-  Button,
-  Hidden,
-  IconButton,
-  makeStyles,
-  Toolbar,
-  Typography,
-} from "@material-ui/core";
-import React, { useEffect, useState } from "react";
-import MenuIcon from "@material-ui/icons/Menu";
-import { Link } from "react-router-dom";
-import SearchBar from "material-ui-search-bar";
-import LinearProgress from "@material-ui/core/LinearProgress";
+         Avatar,
+         Box,
+         Button,
+         Hidden,
+         IconButton,
+         makeStyles,
+         Toolbar,
+         Typography,
+       } from "@material-ui/core";
+       import React, { useEffect, useState } from "react";
+       import MenuIcon from "@material-ui/icons/Menu";
+       import { Link } from "react-router-dom";
+       import SearchBar from "material-ui-search-bar";
+       import LinearProgress from "@material-ui/core/LinearProgress";
+       import { useSelector } from "react-redux";
```

Comment:
Please revert this.

Reply:
@Abhishek-kumar09 Sir , update the desired file in #352.
Kindly check it once.